teerex/sgx-verify/fuzz/fuzz_targets/decode_quote.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+
+use codec::{Decode, Encode};
+use libfuzzer_sys::fuzz_target;
+use sgx_verify::DcapQuote;
+
+fuzz_target!(|data: &[u8]| {
+	let mut copy = data;
+	let quote: Result<DcapQuote, codec::Error> = Decode::decode(&mut copy);
+	//assert!(quote.is_err());
```

Comment:
I think this should not be commented out.

Reply:
Good point, but the this needs to be commented, because the fuzzer actually manages to find byte patterns that can be deserialized. This is not so difficult as there are basically no checks at this point. 
I added a comment that should clarify this.
The test is to check for panics in `decode()`.